application/app/styles/map-styles/map-style.json

```
@@ -0,0 +1,1198 @@
+{
+  "version": 8,
+  "name": "Positron",
+  "metadata": {
+    "mapbox:autocomposite": false,
+    "mapbox:groups": {
+      "101da9f13b64a08fa4b6ac1168e89e5f": {
+        "collapsed": false,
+        "name": "Places"
+      },
+      "a14c9607bc7954ba1df7205bf660433f": {"name": "Boundaries"},
+      "b6371a3f2f5a9932464fa3867530a2e5": {
+        "collapsed": false,
+        "name": "Transportation"
+      }
+    },
+    "mapbox:type": "template",
+    "openmaptiles:mapbox:owner": "openmaptiles",
+    "openmaptiles:mapbox:source:url": "mapbox://openmaptiles.4qljc88t",
+    "openmaptiles:version": "3.x",
+    "maputnik:renderer": "mlgljs"
+  },
```

Si c'est possible je supprimerai les références à `mapbox`, voir si ça ne pose pas de problèmes avec Maputnik.
A priori non : https://maplibre.org/maplibre-style-spec/root/#metadata

```suggestion
    "maplibre:groups": {
      "101da9f13b64a08fa4b6ac1168e89e5f": {
        "collapsed": false,
        "name": "Places"
      },
      "a14c9607bc7954ba1df7205bf660433f": {"name": "Boundaries"},
      "b6371a3f2f5a9932464fa3867530a2e5": {
        "collapsed": false,
        "name": "Transportation"
      }
    },
    "openmaptiles:version": "3.x",
    "maputnik:renderer": "mlgljs"
  },
```

Dans le fichier plus bas il faudrait changer `mapbox:group` par `maplibre:group`

---

En effet, j'ai essayé et aucun souci. J'enlève les labels et je push la nouvelle version